pkg/flagset/flagset.go

```diff
@@ -89,7 +89,7 @@ func ServerWithConfig(cfg *config.Config) []cli.Flag {
 		},
 		&cli.StringFlag{
 			Name:        "debug-addr",
-			Value:       "0.0.0.0:9114",
+			Value:       "0.0.0.0:9194",
```

Comment:
make sure to reserve a [port range here](https://github.com/owncloud/ocis/wiki/Ports)

Reply:
Oh I wanted to change that back... But yes we should define a port for the service.